### PR TITLE
fix(tvos): Fix compilation issues due to missing apis in keyForStorageType for tvos (#1670)

### LINKED ISSUE #1670: Version 13.1.0 getFreeDiskStorage breaks tvOS support
## Summary

Since Version 13.1.0 the library does not support tvOS anymore since the getFreeDiskStorage implementation uses unsupported classes on tvOS.
The previous version 13.0.0 works fine.

|             |     |
| ----------- | --- |
| Version     | 13.1.0   |
| Affected OS | tvOS  |
| OS Version  | 18.0   |

## Current behavior

Compile fails on tvOS, as unsupported classes are used.
image

## Expected behavior

Compile works again on tvOS.

### PR DESCRIPTION
**Pull request by adrianrdzv**
## Description

- Fixes #1670. Apis are [not available for tvos](https://developer.apple.com/documentation/foundation/urlresourcekey/checking_volume_storage_capacity).
- Fixes compilation issue: `sysctl` needs to be imported for tvos as well.

## Compatibility

| OS      | Implemented |
| ------- | :---------: |
| iOS     |    ✅     |
| Android |    ✅     |
| Windows |    ✅     |

## Checklist

* [x] I have tested this on a device/simulator for each compatible OS
* [x] I added the documentation in `README.md`
* [x] I updated the typings files (`privateTypes.ts`, `types.ts`)
* [x] I added a sample use of the API (`example/App.js`)

### CODE REVIEW

File: ios/RNDeviceInfo/RNDeviceInfo.m [hunk 1]
```diff
@@ -37,9 +37,7 @@ typedef NS_ENUM(NSInteger, DeviceType) {
 @import CoreTelephony;
 #endif
 
-#if !TARGET_OS_TV
```
Comment (schie) [inline ios/RNDeviceInfo/RNDeviceInfo.m:40]: Why was this removed?

**Review by schie — COMMENTED** [review]
Overall, I think this is fine. Just have the one question to be answered (I'm not up-to-date on tvOS)

File: ios/RNDeviceInfo/RNDeviceInfo.m (hunk 1 above)
Comment (adrianrdzv) [reply ios/RNDeviceInfo/RNDeviceInfo.m:40]: Due to the introduction of [getStartupTime](https://github.com/react-native-device-info/react-native-device-info/blob/master/ios/RNDeviceInfo/RNDeviceInfo.m#L1062) and the usage of `kinfo_proc`. Now tvos also requires this import. I updated the PR description.

**Review by mikehardy — APPROVED** [review]
LGTM - thank you!

**Pull request merged**